docs/cassandra/multi-az.md

```diff
@@ -0,0 +1,51 @@
+Cassandra across multiple availability zones
+============================================
+
+Navigator supports running Cassandra with rack and datacenter-aware
+replication. To deploy this, you must run a `nodePool` in each availability
+zone, and mark each as a separate Cassandra rack.
+
+The `nodeSelector` field of a nodePool allows scheduling the nodePool to a set
+of nodes matching labels. This should be used with a node label such as
+`failure-domain.beta.kubernetes.io/zone`.
```

Comment:
:grey_question: Maybe add a link to kubernetes failure-domain label docs. If there are any.